### DISCUSSION TITLE
Stripe - Axios - NodeJS

### DISCUSSION THREAD

**Discussion by Cabezagithub** [quasarframework/quasar#14411] [General - Components / Directives / etc]
How can I "Redirect" to Stripe CheckOut Payment URL after execute Axios Post to API Rest developed in Nodejs?

**Quasar-VUE**
    const pago = async () => {
      const resultpago = null
      try {
        const resultpago = await api.post('stripe_test/create-checkout-session')
      } catch (error) {
        console.log('Error : ', error)
      }

      if (resultpago) {
        const url = resultpago.response.URL
        router.push({ path: url, replace: true })  //not work
      }
    }

**API Rest - NodeJS**
// Check Out Session
router.post('/create-checkout-session', async (req, res) => {
   const domainURL = process.env.DOMAIN
   const quan = 3
   const session = await stripe.checkout.sessions.create({
     mode: 'payment',
     line_items: [
       {
         price: 'price_1LhAlpEFYo4ca6813G3rVzyB',
         quantity: quan
       },
     ],
     // ?session_id={CHECKOUT_SESSION_ID} means the redirect will have the session ID set as a query param
     success_url: `http://localhost:8080/pagoexitoso`,
     cancel_url: `http://localhost:8080/pagocancelado`
  })
  return res.redirect(303, session.url)
})

- I've got the URL but I do not know How to "redirect" to de Stripe URL CheckOut page, something like this:
 https://checkout.stripe.com/c/pay/cs_test_a1Ch1Yd2UWJTdBzLbFUbqlmdg07veMiKmmxdaZlzPaYCPIL2h4ojwyDOcG#fidkdWxOYHwnPyd1blpxYHZxWjA0SVFWYklAQ1xqMWZkMz00d1VCalxmM21cZ2hVbF9wcHZHVzZRcWF2bDA1UUpiaWB1MG0zaD1cN191XWB1XWZ0XzJAb0JPdmlHSVVAS19CR1I3dXB1U25hNTVQXX9jZGZ0XCcpJ2N3amhWYHdzYHcnP3F3cGApJ2lkfGpwcVF8dWAnPyd2bGtiaWBabHFgaCcpJ2BrZGdpYFVpZGZgbWppYWB3dic%2FcXdwYHgl

Any suggestion will be appreciated.
Really thank.

**Comment by MilosPaunovic** [comment]
I suggest you ask this question on Stripe or Axios repos, as this has not much to do with Quasar itself.

**Reply by Cabezagithub** [reply]
Thank Milos.
